File: app/src/main/java/com/snaptiongame/app/data/models/CaptionSet.java
package com.snaptiongame.app.data.models;


import com.google.gson.annotations.SerializedName;


/**
 * @author Nick Romero
 */

public class CaptionSet {

    @SerializedName(ID)
    public int id;
    @SerializedName(TITLE)
    public String captionSetTitle;
    @SerializedName(ACTIVE)
    public boolean isCaptionSetActive;

    public static final String TITLE = "title";
    public static final String ID = "id";
    public static final String ACTIVE = "active";

    public static final int MAX_CAPTIONS = 10;
    public static final int MIN_CAPTIONS = 0;

    public CaptionSet() {

    }

    public String getSetName() {
        return captionSetTitle;
    }
}

File: app/src/main/java/com/snaptiongame/app/data/models/CaptionSet.kt
package com.snaptiongame.app.data.models


import com.google.gson.annotations.SerializedName


/**
 * @author Nick Romero
 */

class CaptionSet {

    @SerializedName(ID)
    var id: Int = 0
    @SerializedName(TITLE)
    var setName: String? = null
    @SerializedName(ACTIVE)
    var isCaptionSetActive: Boolean = false

    companion object {
        const val TITLE = "title"
        const val ID = "id"
        const val ACTIVE = "active"
        const val MAX_CAPTIONS = 10
        const val MIN_CAPTIONS = 0
    }
}
